app/javascript/gobierto_data/webapp/components/commons/Visualizations.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <perspective-viewer ref="perspective-viewer" />
+  <perspective-viewer v-if="items" ref="perspective-viewer" />
```

Comment:
Since there's the unique element here, perhaps better don't mount the component Visualizations if there's no items. I mean, set this `v-if` in its parent.